### PR TITLE
feat(deps): dynamic renaming of dependency elements with version #127

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #127

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jason-capsule42** [comment]
Moved back to Work in Progress - import Auro-Library once it's a package and call scripts from there.

https://github.com/AlaskaAirlines/auro-library/issues/13

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
All code changes look good, but the javascript examples are not working on the demo

**Comment by Izook** [comment]
> All code changes look good, but the javascript examples are not working on the demo

Not sure if related but I'm seeing this error in the console when I run the demo: 
image

File: src/dropdownVersion.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default '2.9.9'
```
Comment (Izook) [inline src/dropdownVersion.js:1]: Should these end with new line characters?

File: src/inputVersion.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export default '2.14.1'
```
Comment (Izook) [inline src/inputVersion.js:1]: Same with dropdownVersion

File: src/dropdownVersion.js (hunk 1 above)
Comment (jason-capsule42) [reply src/dropdownVersion.js:1]: These files are generated by the versionWriter.js in the auro-library. We can certainly look at doing an update to the script that that adds a new line. 

It has no impact on this repo and shouldn't hold up this release.

File: src/inputVersion.js (hunk 2 above)
Comment (jason-capsule42) [reply src/inputVersion.js:1]: See comment above.

**Comment by jason-capsule42** [comment]
> > All code changes look good, but the javascript examples are not working on the demo
> 
> Not sure if related but I'm seeing this error in the console when I run the dynamic menu api demo: image
> 
> Otherwise changes look good

@Izook This was definitely related. Great catch! doc example updated to work with dynamic tags.

**Review by Izook — APPROVED** [review]
Cookie Monster

**Pull request merged**